proposals/lip-0055.md

```diff
@@ -400,6 +446,26 @@ unsignedBlockHeaderSchema = {
 }
 ```
 
+
+#### Block Signature Calculation
+
+The block header signature is calculated using the `calculateBlockSignature` function. This function returns a bytes value with length `SIGNATURE_LENGTH_BYTES`.
+
+```python
+calculateBlockSignature():
```

Comment:
just wondering, is there a reason not to name this `calculateSignature(block)`.

Reply:
I see that the function requires other inputs too, so maybe we can leave it as is. Or go totally the other way and call it `calculateSignature(block, secretKey, networkIdentifier)`.

Reply:
Indeed, I would leave it like this unless you have a strong opinion